lib/addons/backstage/index.ts

```diff
@@ -0,0 +1,103 @@
+import { Construct } from 'constructs';
+import { HelmAddOn, HelmAddOnUserProps } from "../helm-addon";
+import { dependable, setPath } from "../../utils";
+import { ClusterInfo, Values } from "../../spi";
+import { ICertificate } from "aws-cdk-lib/aws-certificatemanager";
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface BackstageAddOnProps extends HelmAddOnUserProps {
+    version?: string,
```

Comment:
Let's not include properties which come from HelmAddOnUserProps, such as 
```
name,
namespace,
version
```

Reply:
done